src/components/v5/common/CompletedAction/partials/MeatballMenu/MeatballMenu.tsx

```diff
@@ -32,34 +24,7 @@ const MeatballMenu: FC<MeatballMenuProps> = ({
         withAutoTopPlacement: true,
         top: 12,
       }}
-      items={[
```

Comment:
I would actually much rather keep this inside a component, our component APIs are borked because they receive array configs instead of components. We should never be returning JSX from hooks, so I think this can safely be reverted